roughpy_jax/__init__.py

```diff
@@ -0,0 +1,132 @@
+import numpy as np
+import jax
+import jax.numpy as jnp
+import jax.lax as lax
+
+from dataclasses import dataclass
+from typing import NamedTuple
+
+
+try:
+    from . import _rpy_jax_internals
+except ImportError as e:
+    _rpy_jax_internals = None
+    raise ImportError("RoughPy JAX CPU backend is not installed correctly") from e
+else:
+    # FIXME create using register
+    jax.ffi.register_ffi_target(
+        "cpu_dense_ft_fma",
+        _rpy_jax_internals.cpu_dense_ft_fma(),
+        "cpu"
+    )
+
+
+@dataclass
+class TensorBasis:
```

Comment:
I'm not sure if we should give a new TensorBasis class here or just import it from roughpy.compute. It really depends what our strategy for passing the metadata down to the kernels is going to be.